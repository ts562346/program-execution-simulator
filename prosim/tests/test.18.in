test1 8
DOOP 1
DOOP 2
BLOCK 3
LOOP 10
  DOOP 5
  BLOCK 7
END
HALT

