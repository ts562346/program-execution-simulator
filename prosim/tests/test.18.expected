00000: DOOP
00001: DOOP
00003: BLOCK
00006: DOOP
00011: BLOCK
00018: DOOP
00023: BLOCK
00030: DOOP
00035: BLOCK
00042: DOOP
00047: BLOCK
00054: DOOP
00059: BLOCK
00066: DOOP
00071: BLOCK
00078: DOOP
00083: BLOCK
00090: DOOP
00095: BLOCK
00102: DOOP
00107: BLOCK
00114: DOOP
00119: BLOCK
00126: HALT

DOOP count : 12
DOOP time  : 53
BLOCK count: 11
BLOCK time : 73
